aten/src/ATen/native/cuda/ReduceOpsKernel.cu

```diff
@@ -59,7 +59,7 @@ template <typename scalar_t, typename acc_t=scalar_t>
 void prod_kernel_impl(TensorIterator& iter) {
   gpu_reduce_kernel<scalar_t, scalar_t>(iter, func_wrapper<scalar_t> ([]GPU_LAMBDA(acc_t a, acc_t b) -> acc_t {
     return a * b;
-  }), 1);
+  }), true);
```

Comment:
did you mix this up?  Why is the identity of prod `true` but the identity of and `1`?

Reply:
ah! my bad...